things/policies/service.go

```diff
@@ -127,8 +127,21 @@ func (svc service) AddPolicy(ctx context.Context, token string, p Policy) (Polic
 	}
 
 	// If the client has `g_add` action on the object or is the owner of the object, add the policy
-	ar := AccessRequest{Subject: userID, Object: p.Object, Action: "g_add"}
-	if _, err := svc.policies.EvaluateGroupAccess(ctx, ar); err == nil {
+	areq := AccessRequest{Subject: userID, Object: p.Object, Action: "g_add"}
+	if _, err := svc.policies.EvaluateGroupAccess(ctx, areq); err == nil {
+		page := Page{Subject: userID, Object: p.Object, Offset: 0, Limit: 1}
+		pol, err := svc.policies.Retrieve(ctx, page)
```

Comment:
Is this fetch necessary and why don't we use the policy from `EvaluateGroupAccess`?